src/main/java/org/benetech/servicenet/service/impl/SchedulerServiceImpl.java

```diff
@@ -52,6 +54,31 @@
     @Autowired
     private DataImportReportService dataImportReportService;
 
+    @PostConstruct
+    private void loadJobsAndTriggersOnStartupIfNeeded() {
+
+        Scheduler scheduler = schedulerFactoryBean.getScheduler();
+
+        List<BaseJob> jobs = new ArrayList<>();
+        jobs.add(edenDataUpdateJob);
+        jobs.add(uwbaDataUpdateJob);
+        jobs.add(shelterTechDataUpdateJob);
+        jobs.add(smcConnectTaxonomyUpdateJob);
+        jobs.add(edenTaxonomyUpdateJob);
+        jobs.add(uwbaTaxonomyUpdateJob);
+
+        jobs
+            .forEach(job -> {
+                try {
+                    if (scheduler.getTrigger(job.getInitTrigger().getKey()) == null) {
+                        scheduler.scheduleJob(edenDataUpdateJob.getJobDetail(), edenDataUpdateJob.getInitTrigger());
+                    }
+                } catch (SchedulerException e) {
+                    log.error(e.getMessage());
```

Comment:
I think you should log the entire exception `log.error(e.getMessage(), e);`

Reply:
fixed